packages/formation-react/stories/address-block.stories.mdx

```diff
@@ -1,6 +1,6 @@
 import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
 
-<Meta title="Library/Address block" />
+<Meta title="Components/Address block" />
```

Comment:
Should this go under `Designer Documentation`?

Reply:
It doesn't have any designer-facing documentation. It's just an example of what it looks like (and the code example of how to make it happen).